CMakeLists.txt

```diff
@@ -416,21 +401,6 @@ if (LBANN_HAS_ALUMINUM)
   target_link_libraries(lbann PUBLIC ${Aluminum_LIBRARIES})
 endif ()
 
-if (LBANN_HAS_CUDA)
-  target_link_libraries(lbann PUBLIC ${CUDA_LIBRARIES})
-  target_link_libraries(lbann PUBLIC cuda::toolkit)
-  if (LBANN_WITH_NVPROF)
-    target_link_libraries(lbann PUBLIC ${NVTX_LIBRARIES})
-  endif ()
-  target_link_libraries(lbann PUBLIC ${cuBLAS_LIBRARIES})
-
-  if(LBANN_HAS_NCCL2)
-    target_link_libraries(lbann PUBLIC cuda::nccl)
-  endif()
-endif()
-if(LBANN_HAS_CUDNN)
-  target_link_libraries(lbann PUBLIC ${cuDNN_LIBRARIES})
-endif ()
```

Comment:
Are these getting included elsewhere and that is why they are removed from here?

Reply:
They all get added to the cuda::toolkit IMPORTED target.